project/src/components/add-review/add-review.tsx

```diff
@@ -59,10 +40,10 @@ function AddReview({ film, fetchFilm, addReview }: ConnectedComponentProps): JSX
           <nav className="breadcrumbs">
             <ul className="breadcrumbs__list">
               <li className="breadcrumbs__item">
-                <a href="film-page.html" className="breadcrumbs__link">{film.name}</a>
+                <Link className="breadcrumbs__link" to={AppRoute.Film.replace(':id', id.toString())}>{film.name}</Link>
```

Comment:
можно и без приведения типа

```suggestion
                <Link className="breadcrumbs__link" to={AppRoute.Film.replace(':id', id)}>{film.name}</Link>
```

Reply:
Это же TS а не JS
![image](https://user-images.githubusercontent.com/76129369/141191547-eba028c1-521f-4b06-a824-92154608d6da.png)